masterplan_tools/method/blocks/land_use_parameters.py

```diff
@@ -0,0 +1,29 @@
+"""
+Class holding parameters for land use filter parameters is defined here.
+"""
+import geopandas as gpd
+from pydantic import BaseModel, Field, field_validator
+from masterplan_tools.models.geojson import PolygonGeoJSON
+
+
+class LandUseFeatureProperties(BaseModel):
+    id: int | None = None
+
+
+class LandUseParameters(BaseModel):
+    """
+    Parameters used for land use filter
+    """
+
+    landuse: PolygonGeoJSON[LandUseFeatureProperties]
+    """basic landuse geometries"""
+    no_development: PolygonGeoJSON[LandUseFeatureProperties]
+    """territories with restricted development"""
+    buildings: PolygonGeoJSON[LandUseFeatureProperties] = None
```

Comment:
`None` is not a valid default value for a variable of type `PolygonGeoJSON[LandUseFeatureProperties]`